x-pack/test/apm_api_integration/tests/service_overview/instance_details.spec.ts

```diff
@@ -90,15 +95,15 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     { config: 'basic', archives: [archiveName] },
     () => {
       it('handles empty state when instance id not found', async () => {
-        const response = await supertest.get(
-          url.format({
-            pathname: '/internal/apm/services/opbeans-java/service_overview_instances/details/foo',
+        const response = await apmApiClient.readUser({
+          endpoint: `GET /internal/apm/services/opbeans-java/service_overview_instances/details/foo`,
```

Comment:
You could also use the params to add path properties (`serviceName` and `serviceNodeName`) as above. Same for `instances_detailed_statistics.spec.ts`

Reply:
Thanks!